common/components/WalletDecrypt/PrivateKey.tsx

```diff
@@ -23,7 +17,7 @@ interface Validated {
 }
 
 function validatePkeyAndPass(pkey: string, pass: string): Validated {
-  const fixedPkey = fixPkey(pkey);
+  const fixedPkey = stripHexPrefix(pkey);
```

Comment:
Probably better to name this to strippedPrivateKey or something

Reply:
ack

Reply:
From your comment below, I believe henry meant `fixedPkey -> strippedPkey` rather than `stripHexPrefix -> strippedPkey`.

Reply:
Yes, I also he meant to change the variable name. It's minor however, so I'm merging as is.

Reply:
🤦‍♂️ sorry